### ISSUE TITLE
mixed camelCase and snake_case

### ISSUE THREAD

**Issue by jgebal** [rlove/ut3#1]
One of the things that I've found a bit dirty with utPlsql was mixing standards of coding and API.
Do you think it would be possible to start with clean slate and keep consistency on one approach?
the camelCase though tempting isn't really good for PLSQL mainly due to IDE's and code formatters that can easily make code unreadable by change from `areEqual` to `AREEQUAL` or `areequal` depending on IDE and settings.
`are_equal` seems more Oracle way.

**Comment by rlove** [comment]
Yes I think we can get consistent.   Right now I don't like formatting in
my code base and mentioned it in the readme.md file.   I can see your point
about camel case.
